foundation-week-4/t2q1.py

```diff
@@ -0,0 +1,45 @@
+#Task 1
+tries = 3
+account_balance= 100
+pin_code = 4567
+
+def authenticate(pin_code, user_pin_code):
```

Comment:
✅ Impressive work. I like the structure of the code and the approach to making the `authenticate` and `withdrawl` methods small and testable. They are pure functions.
Improvement: Try to make custom exception when user tries to withdraw more than the available balance. Instead of r`aise Exception`, you can raise your business specific exception i.e. `raise InsufficientFundError`